org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/typesystem/xpect/keyboard.wlk.xt

```diff
@@ -0,0 +1,15 @@
+/* XPECT_SETUP org.uqbar.project.wollok.tests.typesystem.xpect.TypeSystemXpectTestCase END_SETUP */
+import wollok.game.*
+
+object keysTest {
+	
+	// XPECT methodType at m1 --> () => Key
+	method m1() {
+		return keyboard.any()
+	}
+
+	// XPECT! methodType at m2 --> ({(Key) => Void}) => Void
+//	method m2(action) {
```

Comment:
Does this fail because of the closure?

Reply:
Yes, all commented tests are because the closure in the parameter. https://github.com/uqbar-project/wollok/issues/1410